apps/wallet-mobile/src/yoroi-wallets/cardano/byron/ByronWallet.ts

```diff
@@ -1465,6 +1463,47 @@ export class ByronWallet implements YoroiWallet {
       isEasyConfirmationEnabled: this.isEasyConfirmationEnabled,
     }
   }
+
+  CIP30getBalance(tokenId = '*'): Promise<CSL.Value> {
```

Comment:
@michaeljscript how about having a `dappTranslator` object that receives a wallet (this) and returns the 30+95 (cardano interface) methods? (avoid starting with capital if not a constructor or component). e.g dapp = yadayada(this) -> accessing thought, wallet.dappTranslator.getBalance() ...